org.serenityos.Ladybird-gtk4.json

```diff
@@ -43,7 +90,7 @@
                     "path": "."
                 }
             ],
-            "config-opts": ["-DSERENITY_SOURCE_DIR=serenity"],
+            "config-opts": [ "-DSERENITY_SOURCE_DIR=serenity", "-CCache.cmake" ],
             "build-options": {
                 "build-args": ["--share=network"]
```

Comment:
We can't remove --share=network unless I figure out how to either get the flatpak builder to copy the downloaded .tgz/zip files into the `Caches/Foo` subdirectories every time, or we add an override flag/mode to the Serenity CMake to say "just assume your downloaded and extracted files are good m8". right now it checks for BOTH the version.txt and the archive file to be present to assume that we're ready to rock and roll.